homeassistant/components/climate/__init__.py

```diff
@@ -68,8 +69,10 @@
     vol.Required(ATTR_AUX_HEAT): cv.boolean,
 })
 SET_TEMPERATURE_SCHEMA = vol.Schema({
-    vol.Optional(ATTR_ENTITY_ID): cv.entity_ids,
     vol.Required(ATTR_TEMPERATURE): vol.Coerce(float),
+    vol.Inclusive(ATTR_TARGET_TEMP_HIGH, 'auto_mode'): vol.Coerce(float),
+    vol.Inclusive(ATTR_TARGET_TEMP_LOW, 'auto_mode'): vol.Coerce(float),
```

Comment:
We should either require one `ATTR_TEMPERATURE` or both `ATTR_TARGET_TEMP_HIGH` and`ATTR_TARGET_TEMP_LOW`. Now the single temp is always required both not always used.